src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_afs_commands.py

```diff
@@ -0,0 +1,380 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import json
+from datetime import datetime, timedelta
+import unittest
+
+from azure.cli.testsdk import ScenarioTest, JMESPathCheckExists, ResourceGroupPreparer, \
+    StorageAccountPreparer
+from azure.mgmt.recoveryservicesbackup.models import StorageType
+
+subscription_id = "da364f0f-307b-41c9-9d47-b7413ec45535"
+unprotected_afs = "clitestafs"
+protected_afs1 = "sarathtestafs"
+protected_afs2 = "afstest1"
+target_afs = "afstest2"
+storage_account = "sarathsa"
+resource_group_name = "sarath-rg"
+vault_name = "sarath-vault"
+
+
+class BackupTests(ScenarioTest, unittest.TestCase):
+    def test_afs_backup_scenario(self):
```

Comment:
Where do you specify your resource group using resourcegrouppreparer? Typically we use it like [here](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_tests.md#sample-4-prepare-a-resource-group-for-a-test).